### PR TITLE
Added Core functionalities

### PR DESCRIPTION
**Pull request by Le-rg-o**
Some documentation on wanted functionalities.

### CODE REVIEW

File: doc/Core functionalities.md [hunk 1]
```diff
@@ -1 +1,36 @@
-Hello
\ No newline at end of file
+# Wanted functionalities for Pithekos
+## Results of an initial thinking (12/2024)
```
Comment (mvahowe) [inline doc/Core functionalities.md:2]: "initial thinking" (no article) or "the initial brainstorming"

File: doc/Core functionalities.md [hunk 2]
```diff
@@ -1 +1,36 @@
-Hello
\ No newline at end of file
+# Wanted functionalities for Pithekos
```
Comment (mvahowe) [inline doc/Core functionalities.md:1]: wanted => required

File: doc/Core functionalities.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+## Results of an initial thinking (12/2024)
+- Resources management
```
Comment (mvahowe) [inline doc/Core functionalities.md:3]: Resource management (no s even though it's plural)

File: doc/Core functionalities.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+    - Search
```
Comment (mvahowe) [inline doc/Core functionalities.md:4]: Search what? I think search has to look different for local and remote, unless we completely merge the two which we can't do given that remote is expected to not exist most of the time.

File: doc/Core functionalities.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+    - Download
+    - Delete
+
+- Project management
+    - Create
+    - Download
+    - Import (offline)
```
Comment (mvahowe) [inline doc/Core functionalities.md:11]: "sideload" is a good word here

File: doc/Core functionalities.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+    - Save (online / offline)
+    - Personal parameters (layout, etc.)
+
+- Collaboration
+    - Discuss (only online)
+    - Export discussions
+
+- Dashboard?
+    - Latest modifications summary
+    - Projects' progress
```
Comment (mvahowe) [inline doc/Core functionalities.md:25]: Grammatically correct, but most people would just say "Project progress"

File: doc/Core functionalities.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+
+- PDF print
+    - Reuse and improve Scribe's version
+
+- Translation checks
+
+- Alignment
+    - Juxtaligner [(more on that)](https://xenizo.fr/en/how-we-made-a-juxtaligner/)
+
+- Audio OBS (in the long run)
```
Comment (mvahowe) [inline doc/Core functionalities.md:36]: Also text OBS and juxtalinear (view and edit)

**Review by mvahowe — CHANGES_REQUESTED** [review]

File: doc/Core functionalities.md (hunk 4 above)
Comment (Le-rg-o) [reply doc/Core functionalities.md:4]: At that question, let's go with "unsure" for now. It's still possible to think about something close to Sunteleo, where the downloadable resources show up when online. A single research can simplify things. We'll have to look into that at some point.

**Pull request merged**